frontend/src/components/Button.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+ProductInfo.propTypes = {
+    onClick: PropTypes.func,
+    text: PropTypes.string.isRequired,
+}
+
+export default function ProductInfo ({onClick, text}) {
+    return(
+        <Button onClick={onClick}>{text}</Button>
+    )
+}
+
+const Button = styled.button`
+    font-size: inherit;
+`
```

Comment:
Why not simply return the styled component? `ProductInfo` as a name for the default export may lead to some headache when debugging.

```suggestion
export default styled.button`
    font-size: inherit;
`
```